tests/phpunit/Mutator/Operator/SpreadTest.php

```diff
@@ -45,7 +45,7 @@ final class SpreadTest extends AbstractMutatorTestCase
     public function test_mutator($input, $expected = null): void
     {
         if (version_compare(PHP_VERSION, '7.4.0') < 0) {
-            $this->markTestSkipped('Spread operator in Array Expression requires PHP 7.4+');
+//            $this->markTestSkipped('Spread operator in Array Expression requires PHP 7.4+');
```

Comment:
I wonder if we can use [PHPUnit's annotations](https://phpunit.readthedocs.io/en/8.3/incomplete-and-skipped-tests.html#incomplete-and-skipped-tests-requires-tables-api) for this check:

```
@requires PHP >= 7.4
```

Reply:
yes, it works, thanks